### ISSUE TITLE
Create a window and opengl context on windows and on linux

### ISSUE THREAD

**Issue by Freakazo** [Progether/GameFromScratch#2]

**Comment by mikusp** [comment]
Why windowTest.cpp uses old and deprecated functions glBegin() and glEnd()? I thought that we will use OpenGL 3 API.

Second question, why do you want to track build directory? It's only temporary and because of that, I put it into .gitignore file. If you want it to be tracked, just remove the relevant line from .gitignore.

**Comment by Freakazo** [comment]
Yes definitely >= OpenGL 3.2, I just quickly wanted to get something to screen that's why I used it. We probably should get a 3.3 core profile from now on.

I just want the folder to be created on git clones, without it actually tracking anything in the folder.

**Comment by rickky008** [comment]
Nice

**Comment by rickky008** [comment]
Hi
